pkg/sqlite/scene.go

```diff
@@ -1615,5 +1616,26 @@ func (qb *SceneStore) FindDuplicates(ctx context.Context, distance int) ([][]*mo
 		}
 	}
 
+	sortByPath(duplicates)
+
 	return duplicates, nil
 }
+
+func sortByPath(scenes [][]*models.Scene) {
+	lessFunc := func(i int, j int) bool {
+		firstPathI := getFirstSortedPath(scenes[i])
+		firstPathJ := getFirstSortedPath(scenes[j])
+		return firstPathI < firstPathJ
+	}
+	sort.SliceStable(scenes, lessFunc)
+}
+
+func getFirstSortedPath(scenes []*models.Scene) string {
+	paths := make([]string, 0)
+	for _, scene := range scenes {
+		paths = append(paths, scene.Path)
+	}
+	sort.Strings(paths)
+	firstPath := paths[0]
+	return firstPath
```

Comment:
Since you only need the first string dont you only need to compare the path and keep the `smallest`? You dont need to sort the whole list.
eg something like (untested)
```golang
	var firstPath string
	for i, scene := range scenes {
		if i == 0 {
			firstPath = scene.Path
		} else {
			if scene.Path < firstPath {
				firstPath = scene.Path
			}
		}

	}
	return firstPath
```
Also in theory you could get an empty list for this function then you would have an issue with the `paths[0]`?

Reply:
Thanks for the review @bnkai.

While I don't think either point was a major issue (we're sorting a very small collection, and in this code path we know the list is not empty), I have nonetheless refactored the code along the lines you suggest and the code is indeed better now.